pinot-integration-tests/src/test/java/org/apache/pinot/integration/tests/ClusterIntegrationTestUtils.java

```diff
@@ -859,7 +859,9 @@ static void testSqlQuery(String pinotQuery, String brokerUrl, org.apache.pinot.c
     BrokerRequest brokerRequest =
         PinotQueryParserFactory.get(CommonConstants.Broker.Request.SQL).compileToBrokerRequest(pinotQuery);
     if (isSelectionQuery(brokerRequest)) { // selection
-      // TODO: compare results for selection queries, w/o order by
+      // TODO: 1. compare results for selection queries, w/o order by.
+      //       2. validate values of multi-value columns.
+      //       3. remove the restriction that order by column has to be in selection clause.
```

Comment:
Can you add the issue number here? thanks

Reply:
I can add the issue number in the commit.